### DISCUSSION TITLE
[Support]: During review in full screen mode the review speed icon on the tool bar is inoperable 0.14b1

### DISCUSSION THREAD

**Discussion by def1149** [blakeblackshear/frigate#11601] [General Support]
### Describe the problem you are having

In review mode open full screen and click the playback speed icon on the toolbar. 
The list of options doesn't open/drop down. 

### Version

0.14b1

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
N/A
```

### FFprobe output from your camera

```shell
N/A
```

### Frigate stats

```json
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

N/A

**Comment by NickM-27** [comment]
Thanks, fixed in https://github.com/blakeblackshear/frigate/pull/11602
